src/Bundle/Resources/config/services.php

```diff
@@ -8,16 +8,29 @@
 use PrismaMedia\Metrics\Bundle\PrismaMediaMetricsBundle;
 use PrismaMedia\Metrics\MetricAggregator;
 use PrismaMedia\Metrics\MetricRenderer;
+use Symfony\Component\DependencyInjection\Exception\LogicException;
+use function function_exists;
+use function sprintf;
 
 return function (ContainerConfigurator $configurator) {
+
+    $functionRef = sprintf('%s\ref', __NAMESPACE__);
+    $functionService = sprintf('%s\service', __NAMESPACE__);
+
+    $function = function_exists($functionService) ? $functionService : $functionRef;
+
+    if (!function_exists($function)) {
+        throw new LogicException(sprintf('Given function "%s()" not exist.', $function));
```

Comment:
The exception could more helpful. Like "function is missing. Symfony version not supported."

Reply:
Thank you for your feedback. I have changed the message and and hope it is ok for you now.